recipes/libs/botan.yaml

```diff
@@ -0,0 +1,52 @@
+inherit: [cpackage, make, install]
+
+metaEnvironment:
+    PKG_VERSION: "3.6.1"
+    PKG_LICENSE: "BSD-2-Clause"
+
+depends:
+    - libs::zlib-dev
+
+    - use: []
+      depends:
+        - libs::zlib-tgt
+
+checkoutSCM:
+    scm: url
+    url: https://botan.randombit.net/releases/Botan-${PKG_VERSION}.tar.xz
+    digestSHA1: e920c3531fa2daf7348e1a668bfdc0f446e10c8b
+    stripComponents: 1
+
+buildToolsWeak: [python3]
+buildVars: [AR, CXX, CXXFLAGS, CPPFLAGS, ARCH]
+buildScript: |
+    mkdir -p build install
+    pushd build
+    $1/configure.py \
+        --cpu=$ARCH \
+        --os=linux \
+        --cxxflags="$CXXFLAGS $CPPFLAGS" \
+        --ldflags="$LDFLAGS" \
```

Comment:
I checked this and those flags are required so that it finds zlib.

Reply:
Ack